.github/workflows/release.yaml

```diff
@@ -2,26 +2,52 @@ name: Release Charts
 
 on:
   push:
-    branches:
-      - main
+    tags:
+      - "v*.*.*"
 
 permissions: write-all
 
+env:
+  REPLICATED_APP: copia
+
 jobs:
-  release:
+  push-to-replicated:
     runs-on: ubuntu-latest
+    outputs:
+      channel-slug: ${{ steps.create-release.outputs.channel-slug }}
+      release-sequence: ${{ steps.create-release.outputs.release-sequence }}
     steps:
-      - name: Checkout
-        uses: actions/checkout@v2
+      - uses: actions/checkout@b4ffde65f46336ab88eb53be808477a3936bae11 # v4.1.1
         with:
-          fetch-depth: 0
+          repository: ${{ github.event.pull_request.head.repo.full_name }}
+          ref: ${{ github.event.pull_request.head.ref }}
+
+      - name: Define Channel Name
```

Comment:
Minor: Consider using reusable workflows for defining channel, update and package helm and creating channel.

Reply:
I centralized the release workflows.